packages/apps/src/app.ts

```diff
@@ -513,4 +514,33 @@ export class App<TPlugin extends IPlugin = IPlugin> {
     const graphResponse = await this.api.bots.token.getGraph(this.credentials);
     this._tokens.graph = new JsonWebToken(graphResponse.access_token);
   }
+
+  protected async getUserToken(
+    channelId: ChannelID,
+    userId: string
+  ): Promise<string | undefined> {
```

Comment:
nit: why does one method have an explicit return type but not the other?

Reply:
good point. let me fix

Reply:
removed to match others